addons/project/models/project_task.py

```diff
@@ -1590,3 +1595,16 @@ def action_redirect_to_project_task_form(self):
             'url': '/web#model=project.task&id=%s&action=%s&view_type=form' % (self.id, self.env.ref('project.action_view_my_task').id),
             'target': 'new',
         }
+
+    @api.model
+    def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True): #TODO: multiple groupby ??
+        # A read_group can not be performed if records are grouped by personal_stage_type_id as it is a computed field.
+        # personal_stage_type_ids behaves like a M2O from the point of view of the user, we therefore use this field instead.
+        if 'personal_stage_type_id' in groupby and (not lazy or groupby[0] == 'personal_stage_type_id'):
```

Comment:
is it possible that there is no groupby?

Reply:
> is it possible that there is no groupby?

If don't think so. In models.py, groupby is documented as a list and, even if not check is performed, it is manipulated as a list since the beginning of the read_group method.
```
groupby = [groupby] if isinstance(groupby, str) else groupby
lazy_groupby = groupby[:1] if lazy else groupby
```